### PR TITLE
feat(api): added events schema

### PR DESCRIPTION
**Pull request by reenphygeorge**

### CODE REVIEW

File: apps/api/prisma/schema.prisma [hunk 1]
```diff
@@ -75,6 +78,20 @@ model OrganizationInvite {
...
 
+model Events {
+  id              String  @id @unique @default(dbgenerated("(gen_random_uuid())::text"))
+  name            String
+  description     String
+  website         String?
+  location        String
+  createdAt       DateTime @default(now())
+  lastDate        DateTime
+  organizationId  String
```
Comment (sreehari2003) [inline apps/api/prisma/schema.prisma:89]: i was wondering what if the event is hosted virtually  ? 
can we add isOffline boolean if the event is offline we will show the location input in front end else we wont , 
thoughts  @DarkPhoenix2704  @reenphygeorge  ?

Comment (reenphygeorge) [reply apps/api/prisma/schema.prisma:89]: what about making location as 'remote' for such events ?

Comment (DarkPhoenix2704) [reply apps/api/prisma/schema.prisma:89]: location is enough. 
If it is online, people can set it as Remote as said by @reenphygeorge

**Review by sreehari2003 — CHANGES_REQUESTED** [review]
remove pnpm file

**Pull request merged**